.circleci/config.yml

```diff
@@ -586,7 +586,7 @@ commands:
       - run:
           name: run integration test
           command: |  # Runs normally if it is not being run in parallel
-            export DOCKSTORE_CACHE_MIGRATIONS=true
+            # export DOCKSTORE_CACHE_MIGRATIONS=true
```

Comment:
Running into an issue with the cached migrations? 
Is this something to watch out for?

Reply:
Yeah, there's something about the migrations in this PR that is causing the migration caching to fail.  If I delete the last two `sql` blocks from the new `changeSet`, the problem goes away.  When I get a little more time, I'll try to figure it out, but for now, it's probably a good idea to turn the caching off...

Reply:
Ok, @coverbeck  I think add some comments

Reply:
Added comments